bundles/org.openhab.binding.ambientweather/src/main/java/org/openhab/binding/ambientweather/internal/model/DevicesJson.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.ambientweather.internal.model;
+
+import java.util.ArrayList;
+
+/**
+ * The {@link DevicesJson} is the JSON object returned
+ * by the Ambient Weather '/v1/devices' command.
+ * Note that there is nothing in the JSON that defines the type (e.g.
+ * WS-1400-IP, WS-8482, etc.) of weather station.
+ *
+ * @author Mark Hilbush - Initial Contribution
+ */
+public class DevicesJson extends ArrayList<DevicesJson.Container> {
+    public class Container {
```

Comment:
public static class.

and: Does it need to be an ArrayList, or does a Collection or List suffice? Generally people here don't like extending a java container. Not so sure if that is true for dto's as well. It is also not necessary. gson allows to (de)serialize to/from a `List<Container>`

Reply:
Done. I think I found an even simpler solution. Not sure why I missed that the first time. 😕